apps/src/gamelab/GameLabSprite.js

```diff
@@ -260,9 +260,10 @@ module.exports.createSprite = function (x, y, width, height) {
 
 /*
  * Copied code from p5play from Sprite() with targeted modifications that
- * use the additional state parameter
+ * use the additional state parameter and use modifyPosition to determine 
+ * whether to modify the sprite's velocity and position.
  */
-var AABBops = function (p5Inst, type, target, callback) {
+var AABBops = function (p5Inst, type, target, callback, modifyPosition=true) {
```

Comment:
It would work on any collision type and I tested this change with poorva's collision level. Once we merge this gamelab test file I can start to add in a set of tests for collisions that use AABBops.